agent/tool-scripts/datalog/oc-datalog

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python3
+
+import sys
+import os
+
+PROG = os.path.basename(sys.argv[0])
+
+try:
+    tool_output_dir = sys.argv[1]
+except IndexError:
+    print(f"{PROG}: missing required tool output directory argument",
+            file=sys.stderr)
+    sys.exit(1)
+
+components = []
+for idx in range(2, len(sys.argv)):
+    components.append(sys.argv[idx])
+if not components:
+    print(f"{PROG}: missing required components argument(s)",
+            file=sys.stderr)
+    sys.exit(1)
+
+from datetime import datetime
+
+def now():
+    return datetime.utcnow().strftime("%Y-%m-%dT%H:%M:%S.%f")
+
+import subprocess
```

Comment:
move to before line 4

Reply:
Will address after `black` and `flake8` are in enforcing mode.